curriculum/challenges/english/14-responsive-web-design-22/learn-css-flexbox-by-building-a-photo-gallery/6153a3952facd25a83fe8083.md

```diff
@@ -7,35 +7,31 @@ dashedName: step-20
 
 # --description--
 
-Create a media query for screens smaller than `800px` in width. In that media query, create a `#gallery img` rule and set the `width` property to `50%`. This will convert your gallery to a two-column layout.
+Because we set `justify-content: center` on the flex container the images will be centered. The `::after` pseudo-element creates an element that is the last child of the selected element. We can use it to add an empty element after the last image. If we give it the same `width` as the images it will push the last image to the left when the gallery is in a two-column layout.
```

Comment:
What is `Because we set justify-content: center on the flex container the images will be centered` used for?

There is a possibility we need to change `after` to `before`, if we decide to change it to `row-reverse`

```suggestion
The `::after` pseudo-element creates an element that is the last child of the selected element. We can use it to add an empty element after the last image. If we give it the same `width` as the images, it will push the last image to the left when the gallery is in a two-column layout.

Reply:
To explain why the last image is not in the position we would like it to be.

But I agree after having shortened the text it feels a little out of place. I still think we need to motivate and more clearly explain why we are adding the pseudo-element.

Reply:
Maybe?

> The `::after` pseudo-element creates an element that is the last child of the selected element. We can use it to add an empty element after the last image. If we give it the same `width` as the images it will push the last image to the left when the gallery is in a two-column layout. Right now, it is in the center because we set `justify-content: center` on the flex container.

Reply:
I have updated it 👍